<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataTab</class>
 <widget class="QWidget" name="dataTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>349</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="dataView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QwtPlot" name="dataPreviewPlot"/>
       </item>
       <item>
        <widget class="QFrame" name="correctionFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="formulaLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;b&gt;Correction&lt;/b&gt; &lt;i&gt;y ' = a y + b + c &lt;/i&gt;(&lt;i&gt;x - x&lt;sub&gt;0&lt;/sub&gt;&lt;/i&gt;)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="offsetValue"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="offsetLabel">
            <property name="text">
             <string>Offset (&lt;i&gt;b&lt;/i&gt;)</string>
            </property>
            <property name="buddy">
             <cstring>offsetValue</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="scalingLabel">
            <property name="text">
             <string>Scaling (&lt;i&gt;a&lt;/i&gt;)</string>
            </property>
            <property name="buddy">
             <cstring>scalingValue</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="scalingValue"/>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="slopeValue"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="slopeLabel">
            <property name="text">
             <string>Slope (&lt;i&gt;c&lt;/i&gt;)</string>
            </property>
            <property name="buddy">
             <cstring>slopeValue</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="xShiftLabel">
            <property name="text">
             <string>&lt;i&gt;x&lt;/i&gt;-Shift (&lt;i&gt;x&lt;sub&gt;0&lt;/sub&gt;&lt;/i&gt;)</string>
            </property>
            <property name="buddy">
             <cstring>xShiftValue</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="xShiftValue"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
